### PR TITLE
Add session chairs

### LINKED ISSUE #128: Add session chairs to website due 2024-10-14
**Request type**: Website change / addition

**Email** jbtevespro@gmail.com

**Purpose** Add session chairs to website

**Content**  https://docs.google.com/document/d/1NoH4uLn1826dIjtoneNdJRdvwbZQsWviPb_DAHY_dJI/edit?usp=sharing 

**Request Drafts** [Google Drive - Request Drafts](https://drive.google.com/drive/folders/16NgJUc6ed3fOWxZssmPrubhf2EdScDC6?usp=share_link)

- [ ] I've informed the requestor that this is complete

### PR DESCRIPTION
**Pull request by jsubida**
Closes #128

## Checklist:

- [ ] (Committee Chairs Only) I have posted the link for the PR in the usrse slack (#usrse-2024-conference-planning-committee) to ask for content reviewers
- [ ] I have previewed changes locally
- [ ] I have updated the README.md if necessary

cc **ADD GITHUB HANDLES HERE**

### CODE REVIEW

**Comment by jsubida** [comment]
Preview: https://output.circle-artifacts.com/output/job/f8e3861c-f72b-4f3b-9061-209b5241117a/artifacts/0/usrse.github.io/usrse24/program/sessions/index.html

**Comment by danielskatz** [comment]
Can the [full program page](https://us-rse.org/usrse24/program/) also be updated to include chairs?

File: pages/program/sessions.md [hunk 1]
```diff
@@ -1690,6 +1704,8 @@ set_last_modified: true
 <a name="session-5"></a>
 ## Session 5: RSE in Action!
 
+Chair: Abbey Roelefs
```
Comment (mrmundt) [inline pages/program/sessions.md:1707]: Typo:
```suggestion
Chair: Abbey Roelofs
```

**Review by mrmundt — CHANGES_REQUESTED** [review]

File: pages/program/program.md [hunk 2]
```diff
@@ -1281,14 +1280,14 @@ tags will be replaced.-->
   <td rowspan=2 height=290 class=xl736507 width=108 style='height:217.5pt;
   width:81pt'>10:30 AM</td>
   <td colspan=2 class=xl786507 width=302 style='width:227pt'><font
-  class="font116507">Session 1A</font><font class="font66507">: <a href="{{ site.baseurl }}/program/sessions/#session-1a">RSE Pedagogy</a>, </font><font
+  class="font116507">Session 1A</font><font class="font66507">: <a href="{{ site.baseurl }}/program/sessions/#session-1a">RSE Pedagogy</a>, </font>Charles Ferenbaugh, <font
   class="font106507">Hopi/Tewa</font></td>
-  <td colspan=2 class=xl1066507 width=284 style='border-left:none;width:213pt'>Session 1B<font class="font56507">:
-  <a href="{{ site.baseurl }}/program/sessions/#session-1b">AI, ML, and Automation</a>, </font><font
+  <td colspan=2 class=xl1066507 width=284 style='border-left:none;width:213pt'><font
+  class="font116507">Session 1B</font><font class="font66507">: <a href="{{ site.baseurl }}/program/sessions/#session-1b">AI, ML, and Automation</a>, </font>Patrick Diehl, <font
```
Comment (jsubida) [inline pages/program/program.md:1286]: @mrmundt Halp. Y bold? https://output.circle-artifacts.com/output/job/e1489516-fcb0-4f0f-b93f-973fd377f14e/artifacts/0/usrse.github.io/usrse24/program/index.html

Comment (mrmundt) [reply pages/program/program.md:1286]: Well isn't that a pickle... I have no idea. Try wrapping it in:
```
<font class="font116507"></font>
```

**Review by mrmundt — APPROVED** [review]

**Pull request merged**